pkg/installer/upgrade.go

```diff
@@ -10,6 +11,15 @@ import (
 	apiv1 "github.com/storageos/kubectl-storageos/api/v1"
 	pluginutils "github.com/storageos/kubectl-storageos/pkg/utils"
 	pluginversion "github.com/storageos/kubectl-storageos/pkg/version"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	errUpgradeAborted = `
+	Upgrade aborted`
+	errWorkloadsExist = `
+	Discovered bound PVC [%s] using StorageOS storageclass [%s].`
```

Comment:
I'd be a little more explicit about what's expected.  Maybe just append `All workloads that rely on StorageOS volumes should be stopped before upgrading StorageOS.  Re-run with --skip-existing-workload-check to ignore.`?